k8s/deploy-storageos/upgrade-helper/README.md

```diff
@@ -0,0 +1,16 @@
+# StorageOS Upgrade Helper
+
+This section aims to help users with the StorageOS upgrade from any version
+of StorageOS to +1.4.
+
+StorageOS 1.4 incorporates metadata changes that require a migration if running
+a previous version of StorageOS. Such a metadata migration is automated and
+fully handled by the StorageOS software. However, is of paramount importance to
```

Comment:
However, it is of paramount importance to

Reply:
solved. Thank you